Solutions/NetClean ProActive/Analytic Rules/NetClean_Sentinel_analytic_rule.yaml

```diff
@@ -0,0 +1,42 @@
+id: 77548170-5c60-42e5-bdac-b0360d0779bb
```

Comment:
@v-sabiraj Do we have sample data for testing this. I can only verify this based on syntax if the same is not present.
Also, please make it a point to have the sample data for every PR which contains KQL. This would enable us to perform effective testing.